docs/src/main/paradox/java/http/routing-dsl/directives/alphabetically.md

```diff
@@ -37,7 +37,7 @@
 |@ref[extractExecutionContext](basic-directives/extractExecutionContext.md)                                           | Extracts the `ExecutionContext` from the @unidoc[RequestContext]                                                                                                                                                           |
 |@ref[extractMatchedPath](basic-directives/extractMatchedPath.md)                                                     | Extracts the already matched path from the @unidoc[RequestContext]                                                                                                                                                         |
 |@ref[extractMaterializer](basic-directives/extractMaterializer.md)                                                   | Extracts the @unidoc[Materializer] from the @unidoc[RequestContext]                                                                                                                                                               |
-|@ref[extractHost](host-directives/extractHostName.md)                                                                | Extracts the hostname part of the Host request header value                                                                                                                                                         |
+|@ref[extractHostName](host-directives/extractHost.md)                                                                | Extracts the hostname part of the Host request header value                                                                                                                                                         |
```

Comment:
There doesn't seem to be any `extractHostName` directive (any more?). I guess we can use `extractHost` now also for Java.

Reply:
ah even better, I didn't realize ;)